### ISSUE TITLE
aligned and non-aligned PDB structures have the same path in validation and testing sets

### ISSUE THREAD

**Issue by yuyuan871111** [pinder-org/pinder#43]
Dear PINDER team,

I recently used PINDER datasets to explore the protein-protein interactions, and I found this issue.
I found that the `holo_receptor` and `aligned_holo_R` have the same paths (similar issue between `holo_ligand` and `aligned_holo_L`), which is not expected. I think `holo_receptor` and `holo_ligand` should keep the original coordinates like `native`, so we can use those to calculate protein-protein interactions. On the other hand, the `aligned_holo_L` and `aligned_holo_R` are superimposed and can be used for exploring conformational differences.

Then, I checked the PDB files in the downloaded dataset finder. I found that the PDBs in the validation set are not aligned, whereas the PDBs in the testing set are aligned.

The current solution (to explore the protein-protein interactions in the testing set) seems to use `native` pdb, including both `R` and `L` chains.

# in testing set
Code to replicate it:
```python
from pinder.core import get_systems

pinder_system_id = "6ikj__A1_Q9I4L6--6ikj__B1_Q9I4L6" # in testing set
for system in get_systems([pinder_system_id]):
    pass

print(system.holo_receptor.filepath)
print(system.aligned_holo_R.filepath)

print(system.holo_ligand.filepath)
print(system.aligned_holo_L.filepath)
```
Results
```
100%|██████████| 1/1 [00:00<00:00,  6.41it/s]
/home/user/.local/share/pinder/2024-02/test_set_pdbs/6ikj__A1_Q9I4L6-R.pdb
/home/user/.local/share/pinder/2024-02/test_set_pdbs/6ikj__A1_Q9I4L6-R.pdb
/home/user/.local/share/pinder/2024-02/test_set_pdbs/6ikj__B1_Q9I4L6-L.pdb
/home/user/.local/share/pinder/2024-02/test_set_pdbs/6ikj__B1_Q9I4L6-L.pdb
```

<img width="1147" height="458" alt="Image" src="https://github.com/user-attachments/assets/0c2f8bf4-2c2a-49ca-8709-a66f92461882" />

# in validation set

```python
from pinder.core import get_systems

pinder_system_id = "1wtd__B1_Q9RPJ3--1wtd__A1_Q9RPJ3" # in validation set
for system in get_systems([pinder_system_id]):
    pass

print(system.holo_receptor.filepath)
print(system.aligned_holo_R.filepath)

print(system.holo_ligand.filepath)
print(system.aligned_holo_L.filepath)
```
Results
```
100%|██████████| 1/1 [00:00<00:00, 12.90it/s]
/home/user/.local/share/pinder/2024-02/pdbs/1wtd__B1_Q9RPJ3-R.pdb
/home/user/.local/share/pinder/2024-02/pdbs/1wtd__B1_Q9RPJ3-R.pdb
/home/user/.local/share/pinder/2024-02/pdbs/1wtd__A1_Q9RPJ3-L.pdb
/home/user/.local/share/pinder/2024-02/pdbs/1wtd__A1_Q9RPJ3-L.pdb
```

<img width="1069" height="475" alt="Image" src="https://github.com/user-attachments/assets/14dd2364-d628-44de-94e6-8f02db75b62d" />

**Comment by danielkovtun** [comment]
Hi @yuyuan871111 , 

It is expected that the individual monomer PDBs for the receptor and ligand chains are not aligned for the test set. See the note in this [FAQ](https://pinder-org.github.io/pinder/faq.html#what-do-holo-apo-native-and-predicted-structures-within-pindersystem-mean)



> holo: bound conformation. Where bound in the pinder context means there are other interacting protein chains in the original RCSB crystal.
> 
> It is generally used to refer to the individual monomer chains that compose dimers. holo-R = Receptor from native, holo-L = Ligand from native
> 
> The only caveat is that for the test split, we transform the coordinates of the monomers using a PCA-based normalization to ensure that there are no biases introduced during evaluation from methods that may be leaky if the input proteins are pre-oriented.
> 
> All of the train split is left un-perturbed.
> 
> PinderSystem loads all of these for convenience, in case you need to make comparisons to the un-transformed version but tucks this into the code rather than the files on disk to ensure fair evaluation across methods.

`aligned_holo_R` and `aligned_holo_L` are constructed via superposition: 
https://github.com/pinder-org/pinder/blob/20487ac403516f453d8be1b0bdc07a9f7a77d9df/src/pinder-core/pinder/core/index/system.py#L179-L197


If you want the ground-truth dimer structure, you can use the `PinderSystem.native` `Structure`, or if you want the individual Receptor and Ligand chains, you can use the `PinderSystem.native_R` and `PinderSystem.native_L`, respectively. 
https://github.com/pinder-org/pinder/blob/20487ac403516f453d8be1b0bdc07a9f7a77d9df/src/pinder-core/pinder/core/index/system.py#L154-L155


Hope that clears things up! Happy to reopen the issue if you have any further questions.
